enterprise/validators.py

```diff
@@ -48,3 +50,22 @@ def validate_image_size(image):
     if config and not image.size <= valid_max_image_size_in_bytes:
         raise ValidationError(
             _("The logo image file size must be less than or equal to %s KB.") % config.valid_max_image_size)
+
+
+def validate_content_filter_fields(content_filter):
+    """
+    Validate particular fields (if present) passed in through content_filter are certain types.
+    """
+    for key in cftypes:
+        if key in content_filter.keys():
+            if not isinstance(content_filter[key], cftypes[key]['type']):
+                raise ValidationError(
+                    "Content filter '%s' must be of type %s" % (key, cftypes[key]['type'])
+                )
+            if cftypes[key]['type'] == list:
+                if not all(isinstance(x, str) for x in content_filter[key]):
```

Comment:
I think you want to use that `subtype` value here, instead of always insisting that it's a string (not that there's any functional difference right now).

Reply:
ah good point! Will update that.